software/clock_setup_using_arduino_code/Makefile

```
@@ -0,0 +1,64 @@
+PRJ_DIR = $(realpath .)
+
+# ============================================================================= ARGS
+PATH_SKETCH	:= $(PRJ_DIR)/clock_setup/clock_setup.ino
+
+PORT		:= /dev/ttyACM0
+
+FILE_CONFIG	:= arduino-cli.yaml
+BOARD_CORE	:= WCH:ch32v
+BOARD_LIST	:= WCH
+FQBN		:= WCH:ch32v:CH32V00x_EVT
+
+# ============================================================================= TARGETS
+
+# setup and install arduino CLI
+setup-arduino-cli:
+	@ echo " "
+	[ -d ~/arduino_cli ] || mkdir -p ~/arduino_cli; \
+		cd ~/arduino_cli; \
+		curl -fsSL https://raw.githubusercontent.com/arduino/arduino-cli/master/install.sh | sh; \
+		echo "========== Verifying installation..."; \
```

Is this layout a standard from somewhere?

 If not, what about something like this (also for all others and also the comments):
```suggestion
		echo "===== Verifying installation... ====="; \
```

---

What about this formatting? This is also used on multiple occasions